Userland/Libraries/LibSQL/Database.cpp

```diff
@@ -197,9 +197,8 @@ ErrorOr<Vector<Row>> Database::match(TableDef& table, Key const& key)
 ErrorOr<void> Database::insert(Row& row)
 {
     VERIFY(m_table_cache.get(row.table().key().hash()).has_value());
-    // TODO Check constraints
```

Comment:
I think this TODO still exists? IIRC Jan was referring to e.g. https://www.sqlite.org/syntax/table-constraint.html, the comment could've used more explanation though

Reply:
Ah that makes a lot more sense, I´ll make sure to clarify those two comments!

Reply:
Comments reinstated.